src/App/pages/RecipesListPage/components/RecipesList/RecipesList.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import Button from 'components/Button';
+import Card from 'components/Card';
+
+import { IRecipeListItem } from 'utils/types';
+import AlarmClockIcon from '../AlarmClockIcon';
+
+import styles from './RecipesList.module.scss';
+
+interface RecipesListProps {
+  recipesList: IRecipeListItem[];
+}
+
+const RecipesList: React.FC<RecipesListProps> = ({ recipesList }) => {
+  const renderCards = (recipeListItems: IRecipeListItem[]) => {
+    const cards = recipeListItems.map((recipeListItem) => {
+      const { id, title, imageSrc, cookingMinutes, ingredients, nutrition } = recipeListItem;
+
+      const captionSlot = (
+        <div className={styles['recipes-list__caption-slot']}>
+          <AlarmClockIcon className={styles['recipes-list__alarm-clock']} />
+          {`${cookingMinutes} minutes`}
+        </div>
+      );
+
+      const subtitle = ingredients.join(' + ');
+
+      const contentSlot = `${Math.round(nutrition)} kcal`;
+
+      const actionSlot = <Button>Save</Button>;
+
+      return (
+        <Link to={`/recipes/${id}`} key={id}>
```

Comment:
можно хранить роуты в одном конфиге, и использовать их в ссылках и в App, чтобы было удобно менять роуты если вдруг будет нужно
```
// config/routes.ts

const ROUTES = {
  recipesDetail: (id: string) => `/recipes/${id}`
};
```
```suggestion
        <Link to={ROUTES.recipeDetail(id)} key={id}>
```

Reply:
Использовал подход с config/routes.ts